sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  extension (it is not `work_group_specific` anymore). intel/llvm#14271
+- Updated
+  [`sycl_ext_oneapi_matrix`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc)
+  extension:
+  - listed 1x64x16 `bfloat16` matrix combination as available on PVC.
+    intel/llvm#13587
+  - clarified the meaning of `joint_matrix_prefetch` template arguments.
+    intel/llvm#13796
+  - added a note about known issue with some CUDA devices. intel/llvm#14178
+- Added revision 2 of
+  [`sycl_ext_intel_matrix`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc)
+  extension which introduces load, store and fill matrix operations with
+  out-of-bounds checks. intel/llvm#11172
+- Updated
+  [`sycl_ext_oneapi_group`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc)
+  extension:
+  - added new functionality to update arguments and ND-range sizes of kernel
+    nodes within a graph. intel/llvm#12486
+  - clarified `enable_profiling` property in `command_graph::finalize`
+    intel/llvm#14067
+  - clarified how graph edges are recorded in a graph with in-order queues.
+    intel/llvm#12916
+  - clarified how code within command-group functions is handled.
+    intel/llvm#13699
+  - clarified interaction between queue properties and graphs. intel/llvm#13681
+  - added `enable_profiling` property. intel/llvm#13088
+- Updated
+  [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc)
+  extension:
+  - added support external semaphores that can take value. intel/llvm#13860
+  - added support for copying `image_mem_handle` between devices via
+    `ext_oneapi_copy`. intel/llvm#12449
+  - added `fetch_image` overload which accepts sampled image and coordinates.
+    intel/llvm#12447
+  - added type hint template argument to `fetch_cubemap` and `sample_cubemap`
+    APIs. intel/llvm#13742
+
+### SYCLcompat
+
+- Added non-`const` `image2d_max` and `image3d_max` getters. intel/llvm#14138
+- Introduced versioning scheme for the library. intel/llvm#14457
+- Enhanced  masked shuffle functions `select_from_sub_group`,
+  `shift_sub_group_left`, `shift_sub_group_right` and `permute_sub_group_by_xor`
+  to support CUDA devices. intel/llvm#13363
+- Restricted `memory_order` argument of `atomic_ref` passed to
+  `experimental::nd_range_barrier` to match supported on a device.
+  intel/llvm#12974 intel/llvm#13641
+
+## Bug Fixes
+
+### SYCL Compiler
+
+- Fixed that using `-fsycl-link-targets` flag would inadvertently trigger some
```

Comment:
```suggestion
- Fixed a bug where using `-fsycl-link-targets` flag would inadvertently trigger some
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706